pkg/apis/apps/v1/channel_types.go

```diff
@@ -11,63 +11,99 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+// Important:
+//  - JSON tags are required for struct fields to be serializable;
+//  - Run "make api" after modifying this file to regenerate the OpenAPI go file.
+//  - Run "make" to regenerate code after modifying this file.
+
 package v1
 
 import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
 var (
-	// KeyChannelSource is namespacedname tells the source of the deployable
+	// KeyChannelSource is the namespaced name for deployable ???
```

Comment:
this one is used to mark the resource(deployable) created by channel so that the created deployable will have the info of it's parent.

Deployable resource:
https://github.com/open-cluster-management/multicloud-operators-deployable

Reply:
I see. Will revise.